src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.RuntimeOnly.cs

```diff
@@ -2952,4 +2952,20 @@ public static void VerifyRestrictionElementForTimeSpanTest()
         Assert.True(element.LastAttribute.Value == expectedAttribute, string.Format("{0}Test failed for wrong output from schema: {0}Expected Output: {1}{0}Actual Output: {2}",
                 Environment.NewLine, baseline, actualOutput));
     }
+
+    [Fact]
+    public static void NullableAttributeTest()
+    {
```

Comment:
Need some negative tests too. By opening up nullable types, as it's an attribute it needs to be a nullable of a primitive or enum. We should have a negative test for a non-primitive nullable struct. System.Collections.Generic.KeyValuePair<TKey, TValue> would be a good candidate to use.

Reply:
It is fine for Generic.KeyValuePair since IsOptionalValue returns false for it. So it will just throw exception.